k8s/eks/testground_install.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
```

Comment:
I had to `chmod +x` this script and cd into the eks folder.

Reply:
So it is working now?

Reply:
No, I created a thread to discuss where I'm at in https://github.com/testground/infra/pull/78/files#r984508952

Reply:
I did it.